Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/server/Server.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/server/Server.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/server/Server.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/server/Server.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,53 +0,0 @@
-package com.example.schach.server;
-
-import java.io.*;
-import java.net.Inet4Address;
-import java.net.ServerSocket;
-import java.net.Socket;
-import java.net.UnknownHostException;
-
-public class Server {
-    private static int PORT = 22;
-    private static boolean isRunning = false;
-
-
-    public void startServer() {
-        boolean running = true;
-        try {
-            System.out.println(Inet4Address.getLocalHost().getHostAddress());
-        } catch (UnknownHostException e) {
-            e.printStackTrace();
-        }
-        try (ServerSocket serverSocket = new ServerSocket(PORT);
-             Socket socket = serverSocket.accept();
-             BufferedReader br = new BufferedReader(new InputStreamReader(socket.getInputStream()));
-             BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(socket.getOutputStream()));) {
-            isRunning = true;
-
-
-                System.out.println(isRunning);
-
-                bw.write("This was written by the server");
-
-        } catch (IOException e) {
-            e.printStackTrace();
-
-        } finally {
-
-        }
-    }
-
-    public static boolean isIsRunning() {
-        return isRunning;
-    }
-
-    public static String getIpAddress() {
-        try {
-
-            return Inet4Address.getLocalHost().getHostAddress();
-        } catch (UnknownHostException e) {
-            e.printStackTrace();
-        }
-        return "";
-    }
-}
Index: ../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/createAGame.fxml
===================================================================
diff --git a/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/createAGame.fxml b/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/createAGame.fxml
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/createAGame.fxml	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.text.Text?>
-
-<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.schach.client.CreateAGame">
-   <children>
-      <Text layoutX="238.0" layoutY="84.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Your IP Address" />
-      <Text fx:id="IpAddress" layoutX="269.0" layoutY="117.0" strokeType="OUTSIDE" strokeWidth="0.0" />
-      <Text fx:id="status" layoutX="258.0" layoutY="174.0" strokeType="OUTSIDE" strokeWidth="0.0" />
-      <Button layoutX="254.0" layoutY="253.0" mnemonicParsing="false" onAction="#startServer" text="Start Server" />
-   </children>
-</AnchorPane>
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/LoginController.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/LoginController.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/LoginController.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/LoginController.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,60 +0,0 @@
-package com.example.schach.client;
-
-import com.example.schach.server.Server;
-import javafx.event.ActionEvent;
-import javafx.fxml.FXML;
-import javafx.fxml.FXMLLoader;
-import javafx.fxml.Initializable;
-import javafx.scene.Scene;
-import javafx.scene.control.TextField;
-import javafx.scene.text.Text;
-import javafx.stage.Stage;
-
-import java.io.IOException;
-import java.net.URL;
-import java.util.ResourceBundle;
-
-public class LoginController implements Initializable {
-
-    public TextField username;
-    public Text status;
-
-    @Override
-    public void initialize(URL url, ResourceBundle resourceBundle) {
-
-    }
-
-    public void joinAGame(ActionEvent actionEvent) throws IOException {
-        if (username.getText().isEmpty()) {
-            System.out.println("join a game : Not valid");
-            status.setText("Please, enter your username");
-        } else {
-
-
-            Stage stage = new Stage();
-            FXMLLoader fxmlLoader = new FXMLLoader(Login.class.getResource("joinAGame.fxml"));
-            Scene scene = new Scene(fxmlLoader.load(), 500, 500);
-            stage.setTitle("Login");
-            stage.setScene(scene);
-            stage.show();
-
-        }
-    }
-
-    public void createAGame(ActionEvent actionEvent) throws IOException {
-        if (username.getText().isEmpty()) {
-            System.out.println("Create a game : Not valid");
-            status.setText("Please, enter your username");
-        } else {
-
-            Stage stage = new Stage();
-            FXMLLoader fxmlLoader = new FXMLLoader(Login.class.getResource("createAGame.fxml"));
-            Scene scene = new Scene(fxmlLoader.load(), 500, 500);
-            stage.setTitle("Login");
-            stage.setScene(scene);
-            stage.show();
-        }
-
-
-    }
-}
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Figure.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Figure.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Figure.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Figure.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,6 +0,0 @@
-package com.example.schach.client;
-
-public class Figure {
-
-    
-}
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/testFile.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/testFile.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/testFile.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/testFile.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,14 +0,0 @@
-package com.example.schach.client;
-
-import java.net.Inet4Address;
-import java.net.UnknownHostException;
-
-public class testFile {
-    public static void main(String[] args) {
-        try {
-            System.out.println(Inet4Address.getLocalHost().getHostAddress());
-        } catch (UnknownHostException e) {
-            e.printStackTrace();
-        }
-    }
-}
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/JoinAGame.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/JoinAGame.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/JoinAGame.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/JoinAGame.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,67 +0,0 @@
-package com.example.schach.client;
-
-import com.example.schach.server.Server;
-import javafx.event.ActionEvent;
-import javafx.fxml.FXMLLoader;
-import javafx.fxml.Initializable;
-import javafx.scene.Scene;
-import javafx.scene.control.Button;
-import javafx.scene.control.TextField;
-import javafx.scene.text.Text;
-import javafx.stage.Stage;
-
-import java.io.IOException;
-import java.net.URL;
-import java.util.ResourceBundle;
-
-public class JoinAGame implements Initializable {
-
-    public TextField ipAddress;
-    public Text status;
-    public Button playButton;
-
-    @Override
-    public void initialize(URL url, ResourceBundle resourceBundle) {
-
-        playButton.setDisable(true);
-
-    }
-
-    public void connectToServer(ActionEvent actionEvent) {
-        if(isValidIpAddress(ipAddress.getText())){
-            Client client = new Client(ipAddress.getText());
-            if(client.canConnect()){
-                client.connectToServer();
-            } else {
-                status.setText("Can't find Ip Address");
-            }
-
-        }
-    }
-
-
-    public void playChess(ActionEvent actionEvent) throws IOException {
-        Stage stage = new Stage();
-        FXMLLoader fxmlLoader = new FXMLLoader(Login.class.getResource("Chessfield.fxml"));
-        Scene scene = new Scene(fxmlLoader.load(), 400, 350);
-        stage.setTitle("Chessfield");
-        stage.setScene(scene);
-        stage.show();
-    }
-
-    private boolean isValidIpAddress(String ipAddress){
-        if(ipAddress.isEmpty()){
-            status.setText("Please enter Ip Address");
-            return false;
-        }
-        String[] temp = ipAddress.split("\\.");
-        if(temp.length == 4 && !ipAddress.equals("localhost")) {
-            status.setText("Connecting...");
-            return true;
-        }else {
-            status.setText("Invalid IP Address");
-        }
-        return false;
-
-    }
-}
Index: ../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/joinAGame.fxml
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/joinAGame.fxml b/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/joinAGame.fxml
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/joinAGame.fxml	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.TextField?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.text.Text?>
-
-<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.schach.client.JoinAGame">
-   <children>
-      <TextField fx:id="ipAddress" layoutX="202.0" layoutY="134.0" promptText="IP Address" />
-      <Button layoutX="238.0" layoutY="207.0" mnemonicParsing="false" onAction="#connectToServer" text="Connect" />
-      <Text fx:id="status" layoutX="159.0" layoutY="265.0" strokeType="OUTSIDE" strokeWidth="0.0" wrappingWidth="255.0" />
-      <Button fx:id="playButton" layoutX="249.0" layoutY="296.0" mnemonicParsing="false" onAction="#playChess" text="Play" />
-   </children>
-</AnchorPane>
Index: ../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/Chessfield.fxml
===================================================================
diff --git a/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/Chessfield.fxml b/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/Chessfield.fxml
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/Chessfield.fxml	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,153 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.scene.image.*?>
-<?import javafx.scene.layout.*?>
-
-<AnchorPane prefHeight="600.0" prefWidth="600.0" style="-fx-background-color: #8B4513;" xmlns="http://javafx.com/javafx/11.0.2" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.schach.client.ChessboardController">
-   <children>
-      <GridPane fx:id="chessBoardView" gridLinesVisible="true" layoutX="1.0" layoutY="1.0" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="520.0" prefWidth="520.0" AnchorPane.bottomAnchor="35.0" AnchorPane.leftAnchor="35.0" AnchorPane.rightAnchor="35.0" AnchorPane.topAnchor="35.0">
-          <columnConstraints>
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-          </columnConstraints>
-          <rowConstraints>
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-          </rowConstraints>
-          <children>
-              <Pane fx:id="pane00" onMouseClicked="#fieldslected00" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" />
-              <Pane fx:id="pane01" onMouseClicked="#fieldslected01" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" />
-              <Pane fx:id="pane02" onMouseClicked="#fieldslected02" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" />
-              <Pane fx:id="pane03" onMouseClicked="#fieldslected03" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" />
-              <Pane fx:id="pane04" onMouseClicked="#fieldslected04" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" />
-              <Pane fx:id="pane05" onMouseClicked="#fieldslected05" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" />
-              <Pane fx:id="pane06" onMouseClicked="#fieldslected06" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" />
-              <Pane fx:id="pane07" onMouseClicked="#fieldslected07" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" />
-              <Pane fx:id="pane10" onMouseClicked="#fieldslected10" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer0_1" fitHeight="65.0" fitWidth="65.0" layoutY="-1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane11" onMouseClicked="#fieldslected11" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer1_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane12" onMouseClicked="#fieldslected12" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer2_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane13" onMouseClicked="#fieldslected13" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer3_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane14" onMouseClicked="#fieldslected14" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer4_1" fitHeight="65.0" fitWidth="65.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane15" onMouseClicked="#fieldslected15" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer5_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane16" onMouseClicked="#fieldslected16" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer6_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane17" onMouseClicked="#fieldslected17" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer7_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane20" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.rowIndex="2" />
-              <Pane fx:id="pane21" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" GridPane.rowIndex="2" />
-              <Pane fx:id="pane22" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" GridPane.rowIndex="2" />
-              <Pane fx:id="pane23" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" GridPane.rowIndex="2" />
-              <Pane fx:id="pane24" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" GridPane.rowIndex="2" />
-              <Pane fx:id="pane25" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" GridPane.rowIndex="2" />
-              <Pane fx:id="pane26" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" GridPane.rowIndex="2" />
-              <Pane fx:id="pane27" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" GridPane.rowIndex="2" />
-              <Pane fx:id="pane30" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="3" />
-              <Pane fx:id="pane31" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="3" />
-              <Pane fx:id="pane32" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="3" />
-              <Pane fx:id="pane33" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="3" />
-              <Pane fx:id="pane34" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="3" />
-              <Pane fx:id="pane35" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="3" />
-              <Pane fx:id="pane36" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="3" />
-              <Pane fx:id="pane37" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="3" />
-              <Pane fx:id="pane40" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.rowIndex="4" />
-              <Pane fx:id="pane41" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" GridPane.rowIndex="4" />
-              <Pane fx:id="pane42" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" GridPane.rowIndex="4" />
-              <Pane fx:id="pane43" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" GridPane.rowIndex="4" />
-              <Pane fx:id="pane44" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" GridPane.rowIndex="4" />
-              <Pane fx:id="pane45" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" GridPane.rowIndex="4" />
-              <Pane fx:id="pane46" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" GridPane.rowIndex="4" />
-              <Pane fx:id="pane47" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" GridPane.rowIndex="4" />
-              <Pane fx:id="pane50" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="5" />
-              <Pane fx:id="pane51" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="5" />
-              <Pane fx:id="pane52" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="5" />
-              <Pane fx:id="pane53" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="5" />
-              <Pane fx:id="pane54" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="5" />
-              <Pane fx:id="pane55" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="5" />
-              <Pane fx:id="pane56" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="5" />
-              <Pane fx:id="pane57" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="5" />
-              <Pane fx:id="pane60" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.rowIndex="6" />
-              <Pane fx:id="pane61" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" GridPane.rowIndex="6" />
-              <Pane fx:id="pane62" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" GridPane.rowIndex="6" />
-              <Pane fx:id="pane63" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" GridPane.rowIndex="6" />
-              <Pane fx:id="pane64" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" GridPane.rowIndex="6" />
-              <Pane fx:id="pane65" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" GridPane.rowIndex="6" />
-              <Pane fx:id="pane66" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" GridPane.rowIndex="6" />
-              <Pane fx:id="pane67" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" GridPane.rowIndex="6" />
-              <Pane fx:id="pane70" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="7" />
-              <Pane fx:id="pane71" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="7" />
-              <Pane fx:id="pane72" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="7" />
-              <Pane fx:id="pane73" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="7" />
-              <Pane fx:id="pane74" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="7" />
-              <Pane fx:id="pane75" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="7" />
-              <Pane fx:id="pane76" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="7" />
-              <Pane fx:id="pane77" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="7" />
-          </children>
-      </GridPane>
-   </children>
-</AnchorPane>
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/ChessboardController.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/ChessboardController.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/ChessboardController.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/ChessboardController.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,16 +0,0 @@
-package com.example.schach.client;
-
-import javafx.scene.image.ImageView;
-
-public class ChessboardController {
-    public ImageView black_bauer4_1;
-    public ImageView black_bauer0_1;
-    public ImageView black_bauer1_1;
-    public ImageView black_bauer2_1;
-    public ImageView black_bauer3_1;
-    public ImageView black_bauer5_1;
-    public ImageView black_bauer6_1;
-    public ImageView black_bauer7_1;
-
-
-}
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Login.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Login.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Login.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Login.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,23 +0,0 @@
-package com.example.schach.client;
-
-import javafx.application.Application;
-import javafx.fxml.FXMLLoader;
-import javafx.scene.Scene;
-import javafx.stage.Stage;
-
-import java.io.IOException;
-
-public class Login extends Application {
-    @Override
-    public void start(Stage stage) throws IOException {
-        FXMLLoader fxmlLoader = new FXMLLoader(Login.class.getResource("Login.fxml"));
-        Scene scene = new Scene(fxmlLoader.load(), 400, 350);
-        stage.setTitle("Login");
-        stage.setScene(scene);
-        stage.show();
-    }
-
-    public static void main(String[] args) {
-        launch();
-    }
-}
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Client.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Client.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Client.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/Client.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,43 +0,0 @@
-package com.example.schach.client;
-
-import java.io.*;
-import java.net.Socket;
-import java.net.UnknownHostException;
-
-public class Client {
-    private String IPADDRESS;
-    private boolean running = true;
-
-    public Client(String IPADDRESS) {
-        this.IPADDRESS = IPADDRESS;
-    }
-
-    public void connectToServer() {
-        try(Socket socket = new Socket(IPADDRESS,22);
-            BufferedReader br = new BufferedReader(new InputStreamReader(socket.getInputStream()));
-            BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(socket.getOutputStream()));)
-         {
-
-             String s = br.readLine();
-             System.out.println(s);
-
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-    }
-
-    public boolean canConnect(){
-        Socket socket;
-        try{
-            socket = new Socket(IPADDRESS, 22);
-            return true;
-        } catch (UnknownHostException e) {
-            e.printStackTrace();
-        } catch (IOException e) {
-            e.printStackTrace();
-        } finally {
-
-        }
-        return false;
-    }
-}
Index: ../Schach_Mattle_Kinzl/src/main/java/module-info.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/module-info.java b/../Schach_Mattle_Kinzl/src/main/java/module-info.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/module-info.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,8 +0,0 @@
-module com.example.schach {
-    requires javafx.controls;
-    requires javafx.fxml;
-
-
-    opens com.example.schach.client to javafx.fxml;
-    exports com.example.schach.client;
-}
Index: ../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/Chessfield.fxml
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/Chessfield.fxml b/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/Chessfield.fxml
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/Chessfield.fxml	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,153 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.scene.image.*?>
-<?import javafx.scene.layout.*?>
-
-<AnchorPane prefHeight="600.0" prefWidth="600.0" style="-fx-background-color: #8B4513;" xmlns="http://javafx.com/javafx/11.0.2" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.schach.client.ChessboardController">
-   <children>
-      <GridPane fx:id="chessBoardView" gridLinesVisible="true" layoutX="1.0" layoutY="1.0" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="520.0" prefWidth="520.0" AnchorPane.bottomAnchor="35.0" AnchorPane.leftAnchor="35.0" AnchorPane.rightAnchor="35.0" AnchorPane.topAnchor="35.0">
-          <columnConstraints>
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-              <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="50.0" />
-          </columnConstraints>
-          <rowConstraints>
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-              <RowConstraints minHeight="10.0" prefHeight="50.0" vgrow="SOMETIMES" />
-          </rowConstraints>
-          <children>
-              <Pane fx:id="pane00" onMouseClicked="#fieldslected00" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" />
-              <Pane fx:id="pane01" onMouseClicked="#fieldslected01" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" />
-              <Pane fx:id="pane02" onMouseClicked="#fieldslected02" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" />
-              <Pane fx:id="pane03" onMouseClicked="#fieldslected03" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" />
-              <Pane fx:id="pane04" onMouseClicked="#fieldslected04" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" />
-              <Pane fx:id="pane05" onMouseClicked="#fieldslected05" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" />
-              <Pane fx:id="pane06" onMouseClicked="#fieldslected06" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" />
-              <Pane fx:id="pane07" onMouseClicked="#fieldslected07" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" />
-              <Pane fx:id="pane10" onMouseClicked="#fieldslected10" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer0_1" fitHeight="65.0" fitWidth="65.0" layoutY="-1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane11" onMouseClicked="#fieldslected11" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer1_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane12" onMouseClicked="#fieldslected12" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer2_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane13" onMouseClicked="#fieldslected13" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer3_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane14" onMouseClicked="#fieldslected14" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer4_1" fitHeight="65.0" fitWidth="65.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane15" onMouseClicked="#fieldslected15" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer5_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane16" onMouseClicked="#fieldslected16" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer6_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane17" onMouseClicked="#fieldslected17" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="1">
-               <children>
-                  <ImageView fx:id="black_bauer7_1" fitHeight="65.0" fitWidth="65.0" layoutX="1.0" layoutY="1.0" pickOnBounds="true" preserveRatio="true">
-                     <image>
-                        <Image url="@../../../../images/black_pawn.png" />
-                     </image>
-                  </ImageView>
-               </children></Pane>
-              <Pane fx:id="pane20" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.rowIndex="2" />
-              <Pane fx:id="pane21" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" GridPane.rowIndex="2" />
-              <Pane fx:id="pane22" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" GridPane.rowIndex="2" />
-              <Pane fx:id="pane23" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" GridPane.rowIndex="2" />
-              <Pane fx:id="pane24" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" GridPane.rowIndex="2" />
-              <Pane fx:id="pane25" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" GridPane.rowIndex="2" />
-              <Pane fx:id="pane26" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" GridPane.rowIndex="2" />
-              <Pane fx:id="pane27" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" GridPane.rowIndex="2" />
-              <Pane fx:id="pane30" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="3" />
-              <Pane fx:id="pane31" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="3" />
-              <Pane fx:id="pane32" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="3" />
-              <Pane fx:id="pane33" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="3" />
-              <Pane fx:id="pane34" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="3" />
-              <Pane fx:id="pane35" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="3" />
-              <Pane fx:id="pane36" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="3" />
-              <Pane fx:id="pane37" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="3" />
-              <Pane fx:id="pane40" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.rowIndex="4" />
-              <Pane fx:id="pane41" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" GridPane.rowIndex="4" />
-              <Pane fx:id="pane42" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" GridPane.rowIndex="4" />
-              <Pane fx:id="pane43" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" GridPane.rowIndex="4" />
-              <Pane fx:id="pane44" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" GridPane.rowIndex="4" />
-              <Pane fx:id="pane45" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" GridPane.rowIndex="4" />
-              <Pane fx:id="pane46" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" GridPane.rowIndex="4" />
-              <Pane fx:id="pane47" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" GridPane.rowIndex="4" />
-              <Pane fx:id="pane50" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="5" />
-              <Pane fx:id="pane51" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="5" />
-              <Pane fx:id="pane52" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="5" />
-              <Pane fx:id="pane53" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="5" />
-              <Pane fx:id="pane54" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="5" />
-              <Pane fx:id="pane55" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="5" />
-              <Pane fx:id="pane56" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="5" />
-              <Pane fx:id="pane57" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="5" />
-              <Pane fx:id="pane60" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.rowIndex="6" />
-              <Pane fx:id="pane61" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="1" GridPane.rowIndex="6" />
-              <Pane fx:id="pane62" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="2" GridPane.rowIndex="6" />
-              <Pane fx:id="pane63" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="3" GridPane.rowIndex="6" />
-              <Pane fx:id="pane64" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="4" GridPane.rowIndex="6" />
-              <Pane fx:id="pane65" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="5" GridPane.rowIndex="6" />
-              <Pane fx:id="pane66" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="6" GridPane.rowIndex="6" />
-              <Pane fx:id="pane67" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="7" GridPane.rowIndex="6" />
-              <Pane fx:id="pane70" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.rowIndex="7" />
-              <Pane fx:id="pane71" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="1" GridPane.rowIndex="7" />
-              <Pane fx:id="pane72" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="2" GridPane.rowIndex="7" />
-              <Pane fx:id="pane73" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="3" GridPane.rowIndex="7" />
-              <Pane fx:id="pane74" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="4" GridPane.rowIndex="7" />
-              <Pane fx:id="pane75" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="5" GridPane.rowIndex="7" />
-              <Pane fx:id="pane76" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: white;" GridPane.columnIndex="6" GridPane.rowIndex="7" />
-              <Pane fx:id="pane77" prefHeight="200.0" prefWidth="200.0" style="-fx-background-color: grey;" GridPane.columnIndex="7" GridPane.rowIndex="7" />
-          </children>
-      </GridPane>
-   </children>
-</AnchorPane>
Index: ../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/joinAGame.fxml
===================================================================
diff --git a/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/joinAGame.fxml b/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/joinAGame.fxml
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/target/classes/com/example/schach/client/joinAGame.fxml	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.TextField?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.text.Text?>
-
-<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.schach.client.JoinAGame">
-   <children>
-      <TextField fx:id="ipAddress" layoutX="202.0" layoutY="134.0" promptText="IP Address" />
-      <Button layoutX="238.0" layoutY="207.0" mnemonicParsing="false" onAction="#connectToServer" text="Connect" />
-      <Text fx:id="status" layoutX="159.0" layoutY="265.0" strokeType="OUTSIDE" strokeWidth="0.0" wrappingWidth="255.0" />
-      <Button fx:id="playButton" layoutX="249.0" layoutY="296.0" mnemonicParsing="false" onAction="#playChess" text="Play" />
-   </children>
-</AnchorPane>
Index: ../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/createAGame.fxml
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/createAGame.fxml b/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/createAGame.fxml
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/resources/com/example/schach/client/createAGame.fxml	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.text.Text?>
-
-<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.schach.client.CreateAGame">
-   <children>
-      <Text layoutX="238.0" layoutY="84.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Your IP Address" />
-      <Text fx:id="IpAddress" layoutX="269.0" layoutY="117.0" strokeType="OUTSIDE" strokeWidth="0.0" />
-      <Text fx:id="status" layoutX="258.0" layoutY="174.0" strokeType="OUTSIDE" strokeWidth="0.0" />
-      <Button layoutX="254.0" layoutY="253.0" mnemonicParsing="false" onAction="#startServer" text="Start Server" />
-   </children>
-</AnchorPane>
Index: ../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/CreateAGame.java
===================================================================
diff --git a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/CreateAGame.java b/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/CreateAGame.java
deleted file mode 100644
--- a/../Schach_Mattle_Kinzl/src/main/java/com/example/schach/client/CreateAGame.java	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
+++ /dev/null	(revision 1125199824ed71113d00e1be87042bb4d6e30f3a)
@@ -1,42 +0,0 @@
-package com.example.schach.client;
-
-import com.example.schach.server.Server;
-import javafx.event.ActionEvent;
-import javafx.fxml.Initializable;
-import javafx.scene.text.Text;
-
-import java.net.URL;
-import java.util.ResourceBundle;
-
-public class CreateAGame implements Initializable {
-    public Text IpAddress;
-    public Text status;
-
-    @Override
-    public void initialize(URL url, ResourceBundle resourceBundle) {
-        setIpAddress();
-        isConnected();
-
-    }
-
-    private void setIpAddress(){
-        IpAddress.setText(Server.getIpAddress());
-    }
-
-    private void isConnected(){
-
-        if(Server.isIsRunning()){
-            status.setStyle("-fx-text-fill: green");
-            status.setText("Connected");
-        } else {
-
-            status.setText("Waiting for Connection");
-            status.setStyle("-fx-text-fill: red");
-        }
-    }
-
-    public void startServer(ActionEvent actionEvent) {
-        Server server = new Server();
-        server.startServer();
-    }
-}
