Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"MavenProjectsManager\">\r\n    <option name=\"originalFiles\">\r\n      <list>\r\n        <option value=\"$PROJECT_DIR$/pom.xml\" />\r\n      </list>\r\n    </option>\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_17\" default=\"true\" project-jdk-name=\"openjdk-17\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision d795058f1d3fd365f9c39849ea67c061a65eb19a)
+++ b/.idea/misc.xml	(date 1642410932359)
@@ -7,7 +7,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="openjdk-17" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="openjdk-17" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
Index: src/main/java/com/example/schach/client/LoginController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.schach.client;\r\n\r\nimport com.example.schach.server.Server;\r\nimport javafx.event.ActionEvent;\r\nimport javafx.fxml.FXML;\r\nimport javafx.fxml.Initializable;\r\nimport javafx.scene.control.TextField;\r\n\r\nimport java.net.URL;\r\nimport java.util.ResourceBundle;\r\n\r\npublic class LoginController implements Initializable {\r\n\r\n    public TextField username;\r\n\r\n    @Override\r\n    public void initialize(URL url, ResourceBundle resourceBundle) {\r\n\r\n    }\r\n\r\n    public void joinAGame(ActionEvent actionEvent) {\r\n\r\n    }\r\n\r\n    public void createAGame(ActionEvent actionEvent) {\r\n\r\n        Server.startServer();\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/schach/client/LoginController.java b/src/main/java/com/example/schach/client/LoginController.java
--- a/src/main/java/com/example/schach/client/LoginController.java	(revision d795058f1d3fd365f9c39849ea67c061a65eb19a)
+++ b/src/main/java/com/example/schach/client/LoginController.java	(date 1642411473081)
@@ -3,9 +3,13 @@
 import com.example.schach.server.Server;
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
 import javafx.fxml.Initializable;
+import javafx.scene.Scene;
 import javafx.scene.control.TextField;
+import javafx.stage.Stage;
 
+import java.io.IOException;
 import java.net.URL;
 import java.util.ResourceBundle;
 
@@ -22,8 +26,12 @@
 
     }
 
-    public void createAGame(ActionEvent actionEvent) {
-
-        Server.startServer();
+    public void createAGame(ActionEvent actionEvent) throws IOException {
+        Stage stage = new Stage();
+        FXMLLoader fxmlLoader = new FXMLLoader(Login.class.getResource("Login.fxml"));
+        Scene scene = new Scene(fxmlLoader.load(), 400, 600);
+        stage.setTitle("Login");
+        stage.setScene(scene);
+        stage.show();
     }
 }
Index: target/classes/com/example/schach/client/Login.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n\r\n<?import javafx.geometry.Insets?>\r\n<?import javafx.scene.control.Button?>\r\n<?import javafx.scene.control.TextField?>\r\n<?import javafx.scene.layout.AnchorPane?>\r\n\r\n<AnchorPane maxHeight=\"-Infinity\" maxWidth=\"-Infinity\" minHeight=\"-Infinity\" minWidth=\"-Infinity\" prefHeight=\"300.0\" prefWidth=\"400.0\" xmlns=\"http://javafx.com/javafx/17\" xmlns:fx=\"http://javafx.com/fxml/1\">\r\n   <Button layoutX=\"129.0\" layoutY=\"199.0\" mnemonicParsing=\"false\" prefHeight=\"37.0\" prefWidth=\"159.0\" text=\"ConnectToServer\" />\r\n   <TextField id=\"IP Address\" layoutX=\"134.0\" layoutY=\"87.0\" promptText=\"IP Address\" />\r\n   <TextField id=\"Port\" layoutX=\"134.0\" layoutY=\"138.0\" promptText=\"Port\" />\r\n   <opaqueInsets>\r\n      <Insets />\r\n   </opaqueInsets>\r\n</AnchorPane>\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/target/classes/com/example/schach/client/Login.fxml b/target/classes/com/example/schach/client/Login.fxml
--- a/target/classes/com/example/schach/client/Login.fxml	(revision d795058f1d3fd365f9c39849ea67c061a65eb19a)
+++ b/target/classes/com/example/schach/client/Login.fxml	(date 1642409699372)
@@ -5,10 +5,10 @@
 <?import javafx.scene.control.TextField?>
 <?import javafx.scene.layout.AnchorPane?>
 
-<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="300.0" prefWidth="400.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-   <Button layoutX="129.0" layoutY="199.0" mnemonicParsing="false" prefHeight="37.0" prefWidth="159.0" text="ConnectToServer" />
-   <TextField id="IP Address" layoutX="134.0" layoutY="87.0" promptText="IP Address" />
-   <TextField id="Port" layoutX="134.0" layoutY="138.0" promptText="Port" />
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="300.0" prefWidth="400.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.schach.client.LoginController">
+   <Button layoutX="57.0" layoutY="199.0" mnemonicParsing="false" onAction="#createAGame" prefHeight="37.0" prefWidth="159.0" text="Create a game" />
+   <Button layoutX="227.0" layoutY="199.0" mnemonicParsing="false" onAction="#joinAGame" prefHeight="37.0" prefWidth="159.0" text="Join a game" />
+   <TextField fx:id="username" layoutX="116.0" layoutY="88.0" promptText="Username" />
    <opaqueInsets>
       <Insets />
    </opaqueInsets>
Index: src/main/java/com/example/schach/client/testFile.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/schach/client/testFile.java b/src/main/java/com/example/schach/client/testFile.java
new file mode 100644
--- /dev/null	(date 1642411195079)
+++ b/src/main/java/com/example/schach/client/testFile.java	(date 1642411195079)
@@ -0,0 +1,14 @@
+package com.example.schach.client;
+
+import java.net.Inet4Address;
+import java.net.UnknownHostException;
+
+public class testFile {
+    public static void main(String[] args) {
+        try {
+            System.out.println(Inet4Address.getLocalHost().getHostAddress());
+        } catch (UnknownHostException e) {
+            e.printStackTrace();
+        }
+    }
+}
